tests/src/GC/Stress/Framework/ReliabilityFramework.cs

```diff
@@ -1688,28 +1745,38 @@ private void TestPreLoader_Process(ReliabilityTest test, string[] paths)
     {
         Console.WriteLine("Preloading for process mode");
 
+        Console.WriteLine("basepath: {0}, asm: {1}", test.BasePath, test.Assembly);
```

Comment:
I left all of this logging in because I usually end up adding it back whenever I use the RF - I'm happy to delete it if it's too ugly.